api/descriptions/defenders/fargate_yaml_post.md

```diff
@@ -0,0 +1,20 @@
+Returns a protected Fargate task definition for a CloudFormation template given an unprotected task definition.
```

Comment:
@ssugandh maybe we should mention here that this is for CloudFormation YAML template
```suggestion
Returns a protected Fargate task definition for a CloudFormation YAML template given an unprotected task definition.
```